Backend/TimeTracker.Test/TimerTests.cs

```diff
@@ -0,0 +1,108 @@
+                ClientName = "Misha",
+                Tags = new List<string>(),
+                Description = "A very cool Project"
+            });
+
+            int projectId = projectCreationResponse.Data;
+
+            GenericResponseDTO<TimerDTO> startTimerResponse = await timerController.StartTimer(new TimerCreateDTO {
+                Notes = "Working on stuff",
+                ProjectId = projectId
+            });
+
+            Assert.IsTrue(startTimerResponse.Success);
+            Assert.IsTrue(startTimerResponse.Data.Notes.SequenceEqual("Working on stuff"));
+            Assert.AreEqual(startTimerResponse.Data.ProjectId, projectId);
+
+            GenericResponseDTO<List<TimerDTO>> getTimersResponse = await timerController.GetAllTimers();
+
+            List<TimerDTO> timers = getTimersResponse.Data;
+
+            Assert.AreEqual(timers.Count, 1);
+            Assert.AreEqual(timers[0].ProjectId, projectId);
+            Assert.IsTrue(timers[0].Notes.SequenceEqual("Working on stuff"));
+            Assert.AreEqual(timers[0].StartTime, startTimerResponse.Data.StartTime);
+
+            GenericResponseDTO<TimeEntryDTO> stopTimerResponse = await timerController.StopTimer(timers[0].Id);
+            DateTime responseGottenTime = DateTime.Now;
+
+            Assert.IsTrue(stopTimerResponse.Success);
+            Assert.IsTrue(stopTimerResponse.Data.ProjectId == projectId);
+            Assert.IsTrue(stopTimerResponse.Data.Notes.SequenceEqual("Working on stuff"));
+            Assert.AreEqual((responseGottenTime - stopTimerResponse.Data.Day).Minutes, 0);
+
+            getTimersResponse = await timerController.GetAllTimers();
+
+            Assert.AreEqual(getTimersResponse.Data.Count, 0);
+        }
+
+        [TestMethod]
+        public async Task GetTimerByIdTest() {
+
+            await TestAuthHelpers.LogInUser(database, configuration, controllers);
+
+            GenericResponseDTO<int> projectCreationResponse = await projectController.CreateProject(new ProjectCreateDTO{
+                ProjectName = "Another Cool Project",
+                ClientName = "Tristam",
+                Tags = new List<string>(),
+                Description = "An even COOLER project :)"
+            });
+
+            int projectId = projectCreationResponse.Data;
+
+            GenericResponseDTO<TimerDTO> startTimerResponse = await timerController.StartTimer(new TimerCreateDTO {
+                Notes = "Working on things",
+                ProjectId = projectId
+            });
+
+            Assert.IsTrue(startTimerResponse.Success);
+
+            GenericResponseDTO<TimerDTO> getTimerByIdResponse = await timerController.GetTimerById(startTimerResponse.Data.Id);
+
+            Assert.IsTrue(getTimerByIdResponse.Success);
+            Assert.AreEqual(getTimerByIdResponse.Data.Id, startTimerResponse.Data.Id);
+            Assert.IsTrue(getTimerByIdResponse.Data.Notes.SequenceEqual("Working on things"));
```

Comment:
I think I saw this somewhere else too, and this is perfectly fine. I just want to let you know you can do `==` here as well. C# is capable of comparing strings unlike some languages 👍

Reply:
oh that's sick, def good to know 👍